ghpc-kenya-pediatric-hiv/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>New Pediatric (Ages 0-14) HIV Infections in Kenya | CSIS Global Health Policy Center</title>
+        <meta name="description" content="">
```

Comment:
Add the content for the description.
```suggestion
<meta name="description" content="New Pediatric (Ages 0-14) HIV Infections in Kenya | CSIS Global Health Policy Center">
```